sktime/forecasting/tests/test_croston.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from sktime.forecasting.croston import Croston
+from sktime.datasets import load_PBS_dataset
+import pytest
+import numpy as np
+
+# test the Croston's Method against the R package
+
+
+@pytest.mark.parametrize(
+    "fh, r_forecast",
+    [
+        (10, 0.8688921),
+    ],
+)
+def test_Croston_against_r_implementation(fh, r_forecast):
```

Comment:
This one looks good, can we also iterate over one or two additional smoothing parameters?

Reply:
Sure!
Shall I type in another method in this test file to test against different smoothing parameters?

Reply:
I'd add it to the pytest parametrisation, perhaps also for different fhs:
```python
@pytest.mark.parametrize("smoothing, fh, r_forecast",  [(0.1, 10, 0.8688921), (0.5, 5, ...), (...), ...])
```
Three different cases in total should be enough.